src/components/Pollquote/index.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react'
+import { css } from 'react-emotion'
+
+interface PullQuoteProps {
+  //     /** Text for the poll quote */
+  text: string
+  //     /** Custom css for poll quote */
```

Comment:
no need for the `//` comments
```suggestion
  /** Custom css for poll quote */
```